src/toolbar/toolbar-title.jsx

```diff
@@ -1,23 +1,7 @@
 import React from 'react';
+import StylePropable from '../mixins/style-propable';
 import getMuiTheme from '../styles/getMuiTheme';
 
-function getStyles(props, state) {
-  const {
-    baseTheme,
-    toolbar,
-  } = state.muiTheme;
-
-  return {
```

Comment:
It was my bad. I did some extra commits when I should have not. I am making the changes in the latest version of master as we are speaking.

I will resubmit the changes.